packages/svelte/tests/runtime-runes/samples/snippet-optional-arguments-defaults/main.svelte

```diff
@@ -0,0 +1,5 @@
+{#snippet snip(a, b = 1, c=2)}
+{a}{b}{c}
+{/snippet}
```

Comment:
we need to handle things that actually _are_ sequence expressions:

```suggestion
{#snippet snip(a, b = 1, c = (2, 3))}
	{a}{b}{c}
{/snippet}
```

Reply:
Yeah didn't think of how you could pass a sequence as a function parameter